mmf/configs/models/unit/defaults.yaml

```diff
@@ -0,0 +1,104 @@
+model_config:
+  unit:
+    detr_ckpt_path: ''  # the initial DETR parameters to start from (this should be a checkpoint file trained w/ DETR package)
```

Comment:
Can we name this something else as well. `unit_ckpt_path` or `ckpt_path`.

Reply:
renamed to `base_ckpt_path`